core/guests.core.js

```diff
@@ -84,26 +88,59 @@ async function saveNewGuestWithEventId(userId, eventId, guestInfo) {
     },
   };
 
+  if (!emailVerified) {
+    const verifyEmail = new VerifyGuestEmail({
+      to: `${newGuest.email}`,
+      html: `Xin chào ${newGuest.email} <br/>
+      Bạn đã đăng ký tham gia sự kiện ${event.name}. <br/>
+      Chúng tôi cần bạn xác nhận lại email. Vui lòng ấn vào đường dẫn sau để xác nhận: <br/>
+      <a href="#">link</a><br/>
+      Chúng tôi sẽ gửi thông tin vé sớm nhất cho bạn`,
+    });
+
+    mailgun.send(verifyEmail);
```

Comment:
maingun service should be include in Mail class. Then 
```
const verifyEmail = new VerifyGuestEmail(...)
verifyEmail.send();
```

Reply:
Solved: https://github.com/EasyEventV2/ee2_api/pull/9/commits/409cebfa64b084cb7802c255ca941b8fb38eb2ed